src/Modules/CalcSetup.lua

```diff
@@ -606,38 +608,150 @@ function calcs.initEnv(build, mode, override, specEnv)
 				-- Slot is a jewel socket, check if socket is allocated
 				if not env.allocNodes[slot.nodeId] then
 					item = nil
-				elseif item and item.jewelRadiusIndex then
-					-- Jewel has a radius, add it to the list
-					local funcList = item.jewelData.funcList or { { type = "Self", func = function(node, out, data)
-						-- Default function just tallies all stats in radius
-						if node then
-							for _, stat in pairs({"Str","Dex","Int"}) do
-								data[stat] = (data[stat] or 0) + out:Sum("BASE", nil, stat)
+				elseif item then
+					item.jewelData.limitDisabled = nil
+					if item.limit and not env.configInput.ignoreJewelLimits then
+						if jewelLimits[item.title] and jewelLimits[item.title] >= item.limit then
+							item.jewelData.limitDisabled = true
```

Comment:
What is this line for?  I don't see anywhere the `limitDisabled` variable is used

Reply:
in src/Classes/PassiveSpec.lua to try and disable historic jewels, also answering the earlier question, no they dont have limits ingame, we give them limits in the unique data, but they dont actually have them